test/stream.spec.ts

```diff
@@ -45,9 +45,9 @@ describe('stream', () => {
     // the window capacities should have refilled via window updates as received data was consumed
 
     // eslint-disable-next-line @typescript-eslint/dot-notation
-    expect(c1['sendWindowCapacity']).to.equal(defaultConfig.initialStreamWindowSize)
+    expect(c1['sendWindowCapacity']).to.be.gte(defaultConfig.initialStreamWindowSize)
```

Comment:
These tests have become non-deterministic after this change, sometimes the window is the default 256KB, sometimes it's been increased to 16MB by the time the test finishes.